### ISSUE TITLE
Consider augmenting records with due date metadata for social science data

### ISSUE THREAD

**Issue by csjx** [NCEAS/metacatui#243]
When a researcher enters a funding award number, we can query the NSF awards database and determine if the award is under a program code for a social science program.  If so, we can record the award date, and make an additional metadata field entry (TBD) that records the "due date" of the data (5 years from the award for Arctic data, others may vary, so this would be a theme-based feature).  This would allow us to prompt ourselves (server side) when data should be submitted, and perhaps notify PIs as well. This may not be limited to social science data, but might be applied to all awards (those with sooner due dates). Discuss this.

**Comment by amoeba** [comment]
I really like this. As we were talking about this (and other enhancements) I was thinking about the *where* or solving this. What do you think about storing less constrained, less dataset-specific information in RDF/XML instead (in the resource map)?

**Comment by mbjones** [comment]
Agreed this is a good idea.  Given the new idea of also tracking new and existing awards so that we proactively send reminders to PIs at appropriate intervals (and long before any data/metadata has been submitted), we might consider whether it makes sense to create a 'project' entry (however that gets represented) that contains metadata about the project, including award number, program, start and end dates, etc.  Then when a submitted dataset is associated with this project, then we already have the needed information.
